index.html

```diff
@@ -0,0 +1,521 @@
+          </h2>
+          <div
+            class="max-w-lg mx-auto mt-10 grid grid-cols-4 items-center gap-x-8 gap-y-10 md:max-w-xl md:grid-cols-6 md:gap-x-10 lg:mx-0 lg:max-w-none lg:grid-cols-5"
+          >
+            <img
+              id="statamic-logo"
+              class="col-span-2 max-h-12 w-full object-contain lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/transistor-logo-gray-900.svg"
+              alt="Transistor"
+              width="158"
+              height="48"
+            />
+            <img
+              id="statamic-logo"
+              class="col-span-2 max-h-12 w-full object-contain lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/reform-logo-gray-900.svg"
+              alt="Reform"
+              width="158"
+              height="48"
+            />
+            <img
+              id="statamic-logo"
+              class="col-span-2 max-h-12 w-full object-contain lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/tuple-logo-gray-900.svg"
+              alt="Tuple"
+              width="158"
+              height="48"
+            />
+            <img
+              id="statamic-logo"
+              class="col-span-2 max-h-12 w-full object-contain md:col-start-2 lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/savvycal-logo-gray-900.svg"
+              alt="SavvyCal"
+              width="158"
+              height="48"
+            />
+            <img
+              id="statamic-logo"
+              class="col-span-2 col-start-2 max-h-12 w-full object-contain md:col-start-auto lg:col-span-1"
+              src="https://tailwindui.com/img/logos/158x48/statamic-logo-gray-900.svg"
+              alt="Statamic"
+              width="158"
+              height="48"
+            />
+          </div>
+        </div>
+      </section>
+      <!-- Scroll Button -->
+      <button
+        onclick="topFunction()"
+        id="myBtn"
+        title="Go to top"
+        class="hidden fixed bottom-20 right-10 z-50 bg-travel-orange text-white font-bold py-2 px-4 rounded-lg hover:bg-orange-600"
+      >
+        Top
+      </button>
+      <!-- Cookie Banner -->
+      <div
+        id="cookie-banner"
+        class="fixed inset-x-0 z-10 bottom-0 flex flex-col justify-between gap-x-8 gap-y-4 p-4 bg-white border-t border-t-travel-orange md:flex-row md:items-center lg:px-8 opacity-0 transition-opacity duration-500 ease-in-out"
+      >
+        <p class="max-w-4xl text-sm text-black">
+          Discover our curated selection of travel essentials, gear, and
+          accessories—an ensemble designed to enhance your journeys and enrich
+          your experiences on the road. Embrace these offerings at your
+          discretion, knowing they are crafted to elevate your adventures with
+          both practicality and pleasure. See our
+          <a
+            href="/privacy-policy.html"
```

Comment:
Wrong link: cookie-policy.html

Reply:
Spoke to you about this, confirmed we will keep the same name as it will under the same part